examples/twr_aloha/control.c

```diff
@@ -74,6 +73,37 @@ static struct uwb_mac_interface _uwb_mac_cbs = (struct uwb_mac_interface){
     .rx_timeout_cb = _rx_timeout_cb,
 };
 
+/* callback to offload printing */
+static void _print_rng_data_cb(void *arg)
+{
+    uwb_core_rng_data_t *rng_data = (uwb_core_rng_data_t *)arg;
+    turo_t ctx;
+    turo_init(&ctx);
+    turo_dict_open(&ctx);
+    turo_dict_key(&ctx, "t");
+    turo_u32(&ctx, rng_data->time);
+    turo_dict_key(&ctx, "src");
+    turo_u32(&ctx, rng_data->src);
+    turo_dict_key(&ctx, "dst");
+    turo_u32(&ctx, rng_data->dest);
```

Comment:
Could be nice to print dest address in hex format using `turo_string` and `sprintf`